e2e/run

```diff
@@ -156,13 +163,17 @@ customize_cluster_yaml() {
 worker:
   nodePools:
   - name: asg1
+    clusterAutoscalerSupport:
+      enabled: true
```

Comment:
can't we run CA on controller nodes? they already have elevated permissions, adding more wont make it much worse